modules/phase_field/test/tests/free_energy_material/tests

```diff
@@ -114,6 +114,7 @@
       input = 'CoupledValueFunctionFreeEnergy.i'
       exodiff = 'CoupledValueFunctionFreeEnergy_out.e'
       detail = 'Using a pretabulated free energy through PiecewiseMultilinear functions'
```

Comment:
Using a pretabulated **free energy**...

Reply:
well I only looked at the PR name, not its content